components/css/partials/_pullQuote.scss

```diff
@@ -0,0 +1,50 @@
+.pullQuote {
```

Comment:
Possibly rename block to `quote`?

Reply:
Maybe since both of these styles are reserved for quotes, we should just keep in a separate block.